posts/Information-as-an-Asset-Through-Business-Intelligence-and-Data-Analytics.md

```diff
@@ -0,0 +1,27 @@
+---
+authors:
+- Rita Barlow
+tags:
+- Data Analytics
+- Business Intelligence
+- Data 
+- Big Data 
+- Information 
+date: 2019-02-05T12:21:50.000Z
+title: "Information as an Asset Through Business Intelligence and Data Analytics"
+image: BI_VS_Data_Analytics.png
```

Comment:
The `image` tag in the metadata is to display the image at the header image. I see you also included the image in the post, so you may want to keep only one of these. If you keep the header image, the full link will be needed.